lib/src/model.dart

```diff
@@ -13,6 +13,22 @@ import 'pubspec.dart';
 
 part 'model.g.dart';
 
+@JsonSerializable(includeIfNull: false)
+
+/// A processed screenshot contains paths with processed images as stored in GCS.
+class ProcessedScreenshot {
+  final String originalImage; // <imagePath>
+  final String webpImage; // gen/<imagePath>.webp
```

Comment:
So if the original file is `myfolder/myfile.png` this becomes `gen/myfolder/myfile.png.webp` is that correct?
Or are we removing the `.png` so that it becomes `gen/myfolder/myfile.webp`? :D

Reply:
We are not removing the orignal. We make a gen folder for all the images we need.

Reply:
So the answer to the first question is yes.